src/index.js

```diff
@@ -28,31 +33,50 @@ const getDynamicTypes = ({ elementsMap, widgets }) => {
   return dynamicTypes;
 };
 
+const getNpmDependenciesTypesDeclarations = dependencies => {
+  const dependenciesNames = Object.keys(dependencies);
+  if (!dependenciesNames.length) return [];
+  return dependenciesNames.map(packageName => ({
+    path: `/dependencies/${packageName}.${dependencies[packageName]}.d.ts`,
+    content: getModuleTypeDeclaration(packageName)
+  }));
+};
+
 module.exports = {
   configPaths: {
     page: `corvid-types/${TS_CONFIG_PATHS.PAGES}`,
     backend: `corvid-types/${TS_CONFIG_PATHS.BACKEND}`,
     public: `corvid-types/${TS_CONFIG_PATHS.PUBLIC}`
   },
   declarations: {
-    page: ({ elementsMap, widgets } = {}) => {
+    page: ({ dependencies, elementsMap, widgets } = {}) => {
```

Comment:
maybe call it npmDependencies (not sure) ?

Reply:
I think I will stick with dependencies because it is a more generic name